cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -144,12 +153,12 @@ public void dismiss() {
         mAnchor.destroyDrawingCache();
         mAnchor.getViewTreeObserver().removeOnPreDrawListener(mPreDrawListener);
         mPopup.dismiss();
-        
```

Comment:
This is the auto changed stuff by Android Studio as you know.
I'll exclude as possible as I can by the final commit.